system/Packages/perc.gadget.comments/sys__UserDependency--cm/gadgets/repository/perc_comments_gadget/perc_comments_gadget.xml

```diff
@@ -101,7 +101,7 @@
             </script>
             <div id="perc-comments-gadget">
                 <div id="perc-comments-gadget-toolbar" class="perc-gadget-toolbar">
-                    <span class="perc-link" id="perc-set-default-moderation-link" style="display:none">Moderation Default</span>
+                    <span class="perc-link" id="perc-set-default-moderation-link" role="button" title="Moderation Default" style="display:none">Moderation Default</span>
```

Comment:
@rgarg24 Doesn't this need tabindex too?

Reply:
@natechadwick I have added tab index also. It is a different bug but still added to this.